### PR TITLE
Vectorize clamp

### PR DESCRIPTION
**Pull request by cpuhrsch**
Timings follow soon.

### CODE REVIEW

File: aten/src/ATen/cpu/vec256/vec256_float.h [hunk 1]
```diff
@@ -49,21 +49,51 @@ template <> class Vec256<float> {
     }
     return b;
   }
-  static Vec256<float> loadu(const void* ptr, int64_t count = size) {
-    if (count == size)
+  static Vec256<float> loadu(const void* ptr, int64_t count = size, int64_t stride = 1) {
+    if (count == size && stride == 1)
       return _mm256_loadu_ps(reinterpret_cast<const float*>(ptr));
+
+// #ifdef __AVX2__
```
Comment (wanchaol) [inline aten/src/ATen/cpu/vec256/vec256_float.h]: shall we clear the comment?

File: test/test_nn.py [hunk 2]
```diff
@@ -1409,6 +1409,7 @@ def test(nonlinearity, *args, **kwargs):
         test('log_softmax', 0)
         test('leaky_relu', 0.2)
 
+    @unittest.skip('testing')
```
Comment (wanchaol) [inline test/test_nn.py]: Are relus failing now if we pass in a nan tensor?

Comment (cpuhrsch) [reply test/test_nn.py]: This is about properly propagating NaNs. If you pass a NaN into the layer, do you get a NaN out. I'll need to look into this more closely.

**Comment by cpuhrsch** [comment]
On NaN propagation:

See https://www.felixcloutier.com/x86/MINPS.html
In particular http://eigen.tuxfamily.org/bz/show_bug.cgi?id=1373
In short: SIMD max/min and std::max/std::min treat NaNs differently

**Comment by SsnL** [comment]
@cpuhrsch @wanchaol what's the status of this PR?

**Comment by cpuhrsch** [comment]
@SsnL - needs profiling

**Comment by SsnL** [comment]
@cpuhrsch sounds good!

**Comment by ezyang** [comment]
@cpuhrsch I'm going to close this for now so it gets out of the "external contrib" queue. Please feel free to reopen when relevant.

**Pull request closed without merging**